cibuild.cmd

```diff
@@ -0,0 +1,140 @@
+@if not defined _echo echo off
+setlocal
+
+:parseArguments
+if "%1"=="" goto doneParsingArguments
+if /i "%1"=="--scope" set SCOPE=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--target" set TARGET=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--host" set HOST=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--skip-bootstrap" set BOOTSTRAP_ONLY=true&& shift && goto parseArguments
+
+:: Unknown parameters
+goto :usage
+
+set CONFIGURATION=Debug-NetCore
+
+:doneParsingArguments
+
+if "%SCOPE%"=="Compile" (
+    set TARGET_ARG=Build
+) else (
+    set TARGET_ARG=BuildAndTest
+)
+
+:: Assign target configuration
+set CONFIGURATION=
+if "%TARGET%"=="CoreCLR" (
+    set CONFIGURATION=Debug-NetCore
+)
+if "%TARGET%"=="Desktop" (
+    set CONFIGURATION=Debug
+)
+
+if not defined CONFIGURATION (
+    echo Unsupported target detected: %TARGET%. Configuring as if for Desktop
```

Comment:
Yeah, I think the ideal behavior is to special case "not specified" and abort on "specified but not understood".